cloud/migrate-to-cloud/hypershift.md

```diff
@@ -218,6 +218,118 @@ To see all available configuration options for Hypershift, use this command:
 docker run -ti timescale/hypershift:0.4 --help
 ```
 
+### Filtering tables and schemas
+
+Hypershift offers filtering at the schema and table level. Filtering is
+enabled through the `include_schemas`, `exclude_schemas`, `include_tables`, and
+`exclude_tables` parameters in hypershift's configuration file. Note that some
+of these options are mutually exclusive.
+
+The following is an example of a hypershift configuration file with schema and
+table exclusions:
+
+```yaml
+source: postgres://postgres:password@source:5432/database
+target: postgres://postgres:password@target:5432/database
+exclude_schemas:
+- public
+exclude_tables:
+- test.an_uninteresting_table
+```
+
+#### Pattern matching
+
+Hypershift does not support pattern matching on schema or table names in either
+inclusion or exclusion rules. Support may be provided for this in a future
+release.
+
+#### Include schemas
+
+The `include_schemas` configuration can be used to provide an array of schemas
+which should be included in the migration. Only objects in these schemas are
+copied.
+
+Example:
+
+```yaml
+include_schemas:
+- public
+- test
+- '"FooBar"'
+```
+
+Note: as with `pg_dump`, if there is a dependency from one schema to an object
+in another schema which is not included, the dependency will be unfulfilled,
+resulting in failure.
+
+Mutually exclusive with: `exclude_schemas`
+
+#### Exclude schemas
+
+The `exclude_schemas` configuration can be used to provide an array of schemas
+which should not be included in the migration. Only objects which are not in
+these schemas will be copied.
+
+Example:
+
+```yaml
+exclude_schemas:
+- public
+- test
+- '"FooBar"'
+```
+
+Note: as with `pg_dump`, if there is a dependency from one schema to an object
+in another schema which is excluded, the dependency remains unfulfilled,
+resulting in failure to migrate.
+
+Mutually exclusive with: `include_schemas`
```

Comment:
```suggestion
This does not work with [include_schemas][FIXME].
```

Reply:
done. I hope that the anchor links will work correctly. We shall see.